vpr/src/draw/search_bar.h

```diff
@@ -13,19 +13,32 @@
 void search_and_highlight(GtkWidget* /*widget*/, ezgl::application* app);
```

Comment:
Put a comment at the top of the file saying what kinds of functions should go in here (searchbar related, and the search bar is for ...).
Move the function level comments from the .cpp file to this .h file (better convention).
Comments about how the function works go in the .cpp file; comments on what it does and how to call it go in the .h